src/mail/editor/SendMailModel.ts

```diff
@@ -550,7 +594,14 @@ export class SendMailModel {
 	}
 
 	getSenderName(): string {
-		return getSenderNameForUser(this.mailboxDetails, this.user())
+		var senderName = null;
+		for(var contact of this.senderContacts) {
+			if(contact.mailAddresses.find(m => m.address == this.senderAddress)) {
+				senderName = `${contact.firstName} ${contact.lastName}`
+				break
+			}
+		}
+		return senderName ?? getSenderNameForUser(this.mailboxDetails, this.user())
```

Comment:
we generally want to avoid using `var`

Reply:
Forgot to clear here, I fixed it through the rebase